tests/rest/multiple_reports.go

```diff
@@ -127,8 +127,8 @@ func expectNumberOfReports(f *frisby.Frisby, response MultipleReportsResponse, e
 
 // expectOkStatus tests whether the response (JSON) contains status attribute set to 'ok'
 func expectOkStatus(f *frisby.Frisby, response MultipleReportsResponse) {
-	if response.Status == "ok" {
-		f.AddError(fmt.Sprintf("Expected error status, but got '%s' instead", response.Status))
+	if response.Status != "ok" {
+		f.AddError(fmt.Sprintf("Expected ok status, but got '%s' instead", response.Status))
```

Comment:
Nice catch!

Reply:
hmm yes, that's better, as (in theory) the status can be anything.